catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectExportTask.java

```diff
@@ -83,6 +87,18 @@ public void exportProjectToExternalStorage() {
 		}
 	}
 
+	private void deleteUndoFile() {
+		File undoCodeFile = new File(projectDir, UNDO_CODE_XML_FILE_NAME);
+
+		if (undoCodeFile.exists()) {
+			try {
+				StorageOperations.deleteFile(undoCodeFile);
+			} catch (IOException exception) {
+				Log.e(TAG, "Deleting undo file failed.", exception);
```

Comment:
if the exception is thrown, the export should get terminated and the user should be informed with a Toast or a Snackbar.
by the way, `deleteFile()` could throw a `SecurityException` which is not an instance of `IOException` but of `RuntimeException`